clients/payment/src/utils/logger.ts

```diff
@@ -0,0 +1,96 @@
+import { print as printQuery } from "@0no-co/graphql.web";
+import { captureException, init } from "@sentry/react";
+import { isNotNullish, isNullish } from "@swan-io/lake/src/utils/nullish";
+import { CombinedError, Operation, OperationContext } from "urql";
+import { env } from "./env";
+import { isCombinedError } from "./urql";
+
+const FORCE_DEV_LOGGING = false;
+
+const ENABLED = env.IS_SWAN_MODE && (process.env.NODE_ENV === "production" || FORCE_DEV_LOGGING);
+
+export const initSentry = () => {
+  if (ENABLED) {
+    init({
+      release: env.VERSION,
+      dsn: "https://632023ecffdc437984c7a53bbb3aa7a6@o427297.ingest.sentry.io/5454043",
```

Comment:
Probably not the right value

Reply:
Where can I find the correct value ?

Reply:
You need to create a new app in Sentry. For the moment, comment the `ENABLED` line and hardcode no

```ts
// TODO: Enable Sentry when going in production
// const ENABLED = env.IS_SWAN_MODE && (process.env.NODE_ENV === "production" || FORCE_DEV_LOGGING);
const ENABLED = false;

export const initSentry = () => {
  if (ENABLED) {
    init({
      release: env.VERSION,
      dsn: "TODO",
      environment: env.PAYMENT_URL.includes("preprod")
        ? "preprod"
        : env.PAYMENT_URL.includes("master")
        ? "master"
        : "prod",
      normalizeDepth: 5,
    });
  }
};
```